src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -67,7 +68,13 @@ public int ScaleFactorLog
         private readonly DynamoModel model;
         private Point transformOrigin;
         private bool showStartPage = false;
-        
+
+        private ObservableCollection<SimpleViewModel<string>> defaultPythonEngineOptions = new ObservableCollection<SimpleViewModel<string>>();
+        public ObservableCollection<SimpleViewModel<string>> DefaultPythonEngineOptions
```

Comment:
why the need for the field?

Reply:
I didn't want to initialize this more than once.